sql-aggregates/total-dvds.sql

```diff
@@ -0,0 +1,4 @@
+SELECT "storeId",
+  count(*) as "totalDVDs"
+  FROM "inventory"
+  GROUP BY "storeId";
```

Comment:
There should be one total count, not one count per store:

---

![image](https://user-images.githubusercontent.com/16143771/197267430-fed489f2-2ae1-4473-aa9d-0f0c7432adcd.png)